docs/content/how-to/performance-profiling.md

```diff
@@ -0,0 +1,83 @@
+# Manage node-level performance profiling with Performance Profile Controller
+
+## Intro
+
+The `Performance Profile controller`, formerly known as `Performance-Addon Operator`, is now part of `Node Tuning Operator`.
+[Performance Profile controller](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_controller.md) allows you to optimize nodes in your hosted cluster for applications sensitive to CPU and network latency.
+
+In HyperShift, node profiling can be configured by creating ConfigMaps which contain [`PerformanceProfile`](https://docs.openshift.com/container-platform/4.11/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html) objects, and referencing these ConfigMaps in your NodePools.
+
+## Steps
+
+1. Create a ConfigMap which contains a valid `PerformanceProfile` manifest and reference it in a NodePool.
+   The example `PerformanceProfile` manifests defines many profiling parameters like which cpus could be used for workload pods or which ones only for house keeping activities, also enforce a `single-numa-node` topology, etc.
```

Comment:
I'm not sure what you mean by "profiling" in this context. Here is an alternative wording which may be more clear. It might be nice if you could add more explanation for the other settings here as well.
```suggestion
   The example `PerformanceProfile` manifest defines several parameters like how to partition your CPUs into housekeeping and workload partitions, what topology policy to use, and more.
```